shared/src/commonMain/kotlin/com.jetbrains.spacetutorial.entity/Entity.kt

```diff
@@ -0,0 +1,41 @@
+package com.jetbrains.spacetutorial.entity
+
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toInstant
+import kotlinx.datetime.toLocalDateTime
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class RocketLaunch(
+    @SerialName("flight_number")
+    val flightNumber: Int,
+    @SerialName("name")
+    val missionName: String,
+    @SerialName("date_utc")
+    val launchDateUTC: String,
+    @SerialName("details")
+    val details: String?,
+    @SerialName("success")
+    val launchSuccess: Boolean?,
+    @SerialName("links")
+    val links: Links
+) {
+    var launchYear = launchDateUTC.toInstant().toLocalDateTime(TimeZone.UTC).year
```

Comment:
I wonder if this could be done at the serialization level, but I'm not familiar enough with the APIs there to say if it can be, or how complicated it might be